binder/environment.yml

```diff
@@ -10,4 +10,5 @@ dependencies:
   - pytables
   - bokeh
   - rise
-  - larray=0.28
+  - pip:
+    - larray == 0.28
```

Comment:
Why did you fix the version? Is this necessary? The pypi release should not suffer from the conda-forge lag... And if it turns out to be necessary, we need to update the release script to update this version number automatically, otherwise we will forget to update it.

Reply:
No, it is not necessary. I just fix the version as an additional security. I will remove this.

Also, I think tutorial notebooks should be exported to a new dedicated repository in larray-project.

Reply:
what would we gain by moving the tutorial to a separate repository?

Reply:
1. readthedocs takes a huge time to build the API Reference section. This is very annoying when we update the tutorial and want to check how it is rendered.
2. build a myBinder repo with only tutorial notebooks.
3. Have a cleaner conf.py in larray.

Reply:
On the downside, there is a slightly higher risk of forgetting to update the tutorial when a new version requests it when it is in a separate repository, but the upside seems to be larger.